### PR TITLE
Website footer fully responsive

### PR DESCRIPTION
**Pull request by Ryanjso**
Pull request for the website [footer](https://www.figma.com/file/424xbVtVTEeIj9t49NoqDD/Vital-Brand-Guide?node-id=1%3A58). I was unable to test it on my phone, for some reason I can test other servers just not the gatsby one. Maybe something is running on that port already, idk. Maybe Sreya and I can test on her phone during the working sesh tomorrow. 

For the mailchimp integration in the footer, I don't have the mailchimp ids/values as items in Contentful, so we may want to change that. The thing is they're kinda hard to find, you have to like hunt through the page source of a default mailchimp form and there's a lot of inputs that look similar. I'm thinking if they ever want to change it they'll probably have to ask a dev anyways. Also, we could just add to Contentful

### CODE REVIEW

File: src/components/footer/footer-styles.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
+import styled from "styled-components"
+import { white, navyBlue, royalBlue } from "../../constants/colors"
+import footerHex from "../../images/footer-hex.svg"
+import { fonts, fontWeights } from "../../constants/typography"
+import { size } from "../../constants/devices"
+
+export const FooterContainer = styled.div`
+  background-color: ${navyBlue};
+  padding: 40px 100px 40px 100px;
+  font-family: "Work Sans", sans-serif;
```
Comment (sreya-kata) [inline src/components/footer/footer-styles.js]: I think the font they use for the footer is Futura (either the medium one or Futura book). But it should be the same as the navbar

File: src/components/footer/footer-styles.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+  color: white;
+  column-count: 2;
+  margin-top: auto;
+  @media only screen and (max-width: ${size.laptop}px) {
+    column-count: 1;
+  }
+`
+
+export const FooterLink = styled.a`
+  font-family: "Work Sans", sans-serif;
```
Comment (sreya-kata) [inline src/components/footer/footer-styles.js]: ^You can also abstract the font out into the `typography.js` file so we can reference the footer font variable here

File: src/components/footer/footer.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        </FooterSubsectionContainer>
+        <FooterSubsectionContainer>
+          <FooterScroll onClick={() => scrollTo("#scrollTop")}>
+            Back To Top ↑
+          </FooterScroll>
+          <FooterContactContainer>
+            <FooterSocialBlock>
+              {data.contentfulFooter.socialMedia.map(social => {
+                return (
+                  <a href={social.link}>
```
Comment (sreya-kata) [inline src/components/footer/footer.js]: can you make these links `target="_blank"` so it opens in a new tab?

File: src/page-styles 2/what-we-do-styles.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components"
```
Comment (sreya-kata) [inline src/page-styles 2/what-we-do-styles.js]: I think this is a duplicate folder lol. "page-styles" and "page-styles 2"

**Review by sreya-kata — COMMENTED** [review]
Looks really good! Just some small stylistic stuff
Here's a pic from my phone (the mobile version works):
mlbile

File: src/page-styles 2/what-we-do-styles.js (hunk 4 above)
Comment (Ryanjso) [reply src/page-styles 2/what-we-do-styles.js]: ugh why does my stupid thing keep duplicating files 😭

File: src/components/footer/footer-styles.js (hunk 1 above)
Comment (josephmannis) [reply src/components/footer/footer-styles.js]: Can you pull from the variables here either way please?

File: src/components/footer/footer-styles.js [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+`
+
+export const FooterSubsectionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+`
+
+export const FooterDescription = styled.h3`
+  font-family: "Work Sans", sans-serif;
```
Comment (josephmannis) [inline src/components/footer/footer-styles.js]: Here too please

File: src/components/footer/footer-styles.js [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+  width: 40px;
+  margin-left: 12px;
+
+  @media only screen and (max-width: ${size.tablet}px) {
+    margin: 0 0 12px 0;
+  }
+`
+
+export const ConnectContactText = styled.p`
+  font-family: "Work Sans", sans-serif;
```
Comment (josephmannis) [inline src/components/footer/footer-styles.js]: Type

File: src/components/footer/footer-styles.js [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+  @media only screen and (max-width: ${size.tablet}px) {
+    margin: 0;
+  }
+`
+
+export const MailtoLink = styled.a`
+  line-height: 20px;
+  margin-left: 16px;
+  text-decoration: none;
+  font-family: "Work Sans", sans-serif;
```
Comment (josephmannis) [inline src/components/footer/footer-styles.js]: Type

File: src/components/footer/footer.js (hunk 3 above)
Comment (josephmannis) [inline src/components/footer/footer.js]: Add `keys` as well please

File: src/components/footer/footer.js [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+              ...GatsbyContentfulFluid
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  return (
+    <FooterContainer>
```
Comment (josephmannis) [inline src/components/footer/footer.js]: Can you add `role="img"` and `aria-label="<description of the polygon image"` please? For background image accessibility.

File: src/components/teampage/advisor-member.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+import { AdvisorCard, AdvisorImage, AdvisorName } from "./advisor-member-styles"
+import { MemberPosition, Overlay, OverlayText } from "./team-shared-styles"
+
+export default ({ name, photo, position, linkedinUrl }) => (
+  <AdvisorCard>
+    <AdvisorImage photoUrl={photo.fluid.src}>
```
Comment (josephmannis) [inline src/components/teampage/advisor-member.js]: Can you please ensure this has alt text?

File: src/components/teampage/advisors.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+import { AdvisorContainer, AdvisorsTitle } from "./advisors-styles.js"
+import AdvisorMember from "../teampage/advisor-member"
+
+export default ({ advisors }) => (
+  <div>
+    <AdvisorsTitle>Advisors</AdvisorsTitle>
+    <AdvisorContainer>
+      {advisors.map((adv, index) => {
+        return (
+          <AdvisorMember
```
Comment (josephmannis) [inline src/components/teampage/advisors.js]: Please add a key here

File: src/components/teampage/team-member.js [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+  major,
+  position,
+  linkedinUrl,
+  team,
+  index,
+  isLast,
+}) => (
+  <CardPosition index={index} isLast={isLast}>
+    <ContactCardContainer index={index}>
+      <MemberImg photoUrl={photo.fluid.src}>
```
Comment (josephmannis) [inline src/components/teampage/team-member.js]: Same deal with the `role="img"` stuff

File: src/components/teampage/teams.js [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+  BackgroundHex,
+  TitleHolder,
+} from "../teampage/teams-styles"
+import TeamMember from "../teampage/team-member"
+
+export default ({ teamName, members, index }) => (
+  <TeamContainer>
+    <TitleHolder>
+      <TeamTitle index={index}>{teamName}</TeamTitle>
+      <BackgroundHex index={index}></BackgroundHex>
```
Comment (josephmannis) [inline src/components/teampage/teams.js]: `role="img"` etc

**Review by josephmannis — CHANGES_REQUESTED** [review]
Noticing a body issue here on smaller devices. It's probably something to do with the content on the page. This appears to be an issue on every page – so if you don't want to fix it here please create an issue for it.

image

Also, can you get rid of the `page-styles 2` folder?

Last thing, there are a lot of team member and other changes in this PR – are those supposed to be complete or is this intended to be just the footer stuff? If so, there doesn't seem to be any data for the `our-team` page which makes it hard to test. Can you either get that data to show or isolate the website footer changes in this PR and pull the team member stuff into another branch?

**Comment by sreya-kata** [comment]
> Noticing a body issue here on smaller devices. It's probably something to do with the content on the page. This appears to be an issue on every page – so if you don't want to fix it here please create an issue for it.
> 
> image
> 
> Also, can you get rid of the `page-styles 2` folder?
> 
> Last thing, there are a lot of team member and other changes in this PR – are those supposed to be complete or is this intended to be just the footer stuff? If so, there doesn't seem to be any data for the `our-team` page which makes it hard to test. Can you either get that data to show or isolate the website footer changes in this PR and pull the team member stuff into another branch?

I forgot I fixed this in my code (which isn't merged yet) but it's cause of the navbar so we have to add `overflow-x: hidden` in the `nav-wrapper` (but also probably in the `layout.css` for good measure).

**Comment by Ryanjso** [comment]
Ok re-requesting here in a second. Fixed all the stuff here and the weird team inclusion should be handled cause that was merged with main.

**Comment by Ryanjso** [comment]
ooops forgot to add a made with love by scout thing at bottom, will do that now

File: src/constants/typography.js [hunk 13]
```diff
@@ -149,6 +149,44 @@ const Connect = styled.a`
   }
 `
 
+const FooterLink = styled.a`
+  font-size: ${fontSizes.p};
+  font-family: ${fonts.navButton};
```
Comment (sreya-kata) [inline src/constants/typography.js]: I think the font for the links might be a little too bold from the designs? I tried doing `fonts.button` with letter spacing 0.5 and it looks more like what they have:
footer links
We can also increase the mobile line-height to around 24px
mobile footer links

**Review by sreya-kata — COMMENTED** [review]
Looks great!! Just one comment about the font. Also Riha said something about the big "ViTAL" headline being where the logo is but I'm not sure. I'll ask her.

**Review by josephmannis — APPROVED** [review]
Looks good, can you just make the social icons open a new tab? They aren't doing that for me. Otherwise nice!

**Comment by Ryanjso** [comment]
> Looks good, can you just make the social icons open a new tab? They aren't doing that for me. Otherwise nice!

It opens a new one for me in both safari and chrome, so I'm not sure what the prob would be. What device/browser are you testing on?

**Review by sreya-kata — APPROVED** [review]
👀  💖

**Pull request merged**